lua/lvim/core/autocmds.lua

```diff
@@ -99,6 +99,28 @@ function M.load_defaults()
         end,
       },
     },
+    {
+      "ColorScheme",
+      {
+        group = "_lvim_colorscheme",
+        callback = function()
+          if lvim.builtin.breadcrumbs.active then
```

Comment:
I wonder if this is always safe, maybe we should add it in a separate autocmd when we know that `lvim.builtin.breadcrumbs` is available

Reply:
I can't imagine a case where it's not safe, if breadcrumbs is on reload it real quick on colorscheme change, if not ignore it.